runtime/kusama/src/lib.rs

```diff
@@ -1163,6 +1163,8 @@ impl parachains_origin::Config for Runtime {}
 
 impl parachains_configuration::Config for Runtime {
 	type WeightInfo = weights::runtime_parachains_configuration::WeightInfo<Runtime>;
+	type HrmpMaxOutboundChannelsBound = frame_support::traits::ConstU32<128>;
```

Comment:
@pepyakin @rphmeier can either of you confirm that this value is a sensible bound?

Reply:
Yes, 128 seems sensible upper bound. Each parachain can only open a single channel to another parachain. There is only so many parachains, i.e 100.